### PR TITLE
FIX create only mention notifications

### PR DESCRIPTION
**Pull request by makaroni4**

### CODE REVIEW

File: app/models/comment.rb [hunk 1]
```diff
@@ -36,8 +41,11 @@ def observe_post_to_author
   end
   
   def notify_observing
+    mentioned_user_ids = mention_notifications.pluck(:user_id)
+    
     post.observings.includes(:user).each do |observing|
       next if observing.user_id == user_id
+      next if observing.user_id.in?(mentioned_user_ids)
```
Comment (releu) [inline app/models/comment.rb]: or ?

**Comment by releu** [comment]
What about mentioning user in post?

File: app/models/comment.rb (hunk 1 above)
Comment (makaroni4) [inline app/models/comment.rb]: ||

don't like long conditions

File: app/models/comment.rb [hunk 2]
```diff
@@ -36,8 +41,11 @@ def observe_post_to_author
   end
   
   def notify_observing
+    mentioned_user_ids = mention_notifications.pluck(:user_id)
+    
     post.observings.includes(:user).each do |observing|
```
Comment (releu) [inline app/models/comment.rb]: post.observings.where("user_id not in ?", (mentioned_user_ids << user_id)...

**Pull request merged**